docs/tempo/website/operations/deployment.md

```diff
@@ -32,4 +32,45 @@ Helm charts are available in the grafana/helm-charts repo:
 ## Kubernetes manifests
 
 You can find a collection of Kubernetes manifests to deploy Tempo in the [operations/kube-manifests](https://github.com/grafana/tempo/tree/main/operations/kube-manifests) folder.
-These are generated using the Tanka / Jsonnet.
\ No newline at end of file
+These are generated using the Tanka / Jsonnet.
+
+# Deployment Scenarios
+
+Tempo can be deployed in three modes.
+
+* Single Binary
+* Scalable Single Binary
+* Microservices
+
+These modes are controlled by the runtime configuration `target`, or using the CLI flag `-target`.  The default target is `all`, which we refer to as "Single Binary".
+
+## Single Binary
+
+This mode is the simplest to get started, in which all of the top-level components, referenced in the [Architecture](architecture) section, are run within the same instance of Tempo.
+
+## Scalable Single Binary
+
+This mode allows the simplicity of the Single Binary mode, but utilizes `memberlist` in order to cluster the components.  The following configuration is required to use the memberlist.
```

Comment:
i don't think we should mention memberlist explicitly. there are a lot of ways to cluster tempo. let's just talk about how it packages all tempo components into one package that can be scaled horizontally.

also create and link to a docker-compose example that runs in this mode.